packages/flutter_tools/templates/module/android/library/include_flutter.groovy.copy.tmpl

```diff
@@ -1,37 +1,17 @@
-// Generated file. Do not edit.
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
 
 def scriptFile = getClass().protectionDomain.codeSource.location.toURI()
 def flutterProjectRoot = new File(scriptFile).parentFile.parentFile
+def localPropertiesFile = new File(flutterProjectRoot, ".android/localx.properties")
 
-gradle.include ':flutter'
-gradle.project(':flutter').projectDir = new File(flutterProjectRoot, '.android/Flutter')
+def properties = new Properties()
 
-def plugins = new Properties()
-def pluginsFile = new File(flutterProjectRoot, '.flutter-plugins')
-if (pluginsFile.exists()) {
-    pluginsFile.withReader('UTF-8') { reader -> plugins.load(reader) }
-}
+assert localPropertiesFile.exists(), "❗️The Flutter module doesn't have a `$localPropertiesFile` file." +
```

Comment:
FYI @xster. This new error message will be useful when building the host app without running `pub get` in the module.

Reply:
Awesome! Thanks. 

nit: trailing period.

Reply:
Done